Frontend/src/App.css

```diff
@@ -15,18 +15,24 @@
   --lime: #E5E216;
   --sage: #C0C382;
   --orange-red: #D93F24;
-  --default-background: white;
-  --error-color: red;
+
 }
+  --error-color: red;
+
 
 html {
   height: 100%;
 }
 
 body {
   height: 100%;
+  background-image: url("images/background.png");
```

Comment:
For brownie points, grab the svg version of the file and use that. The benefits of svg's are that they are a lot smaller than other image types. That isn't always the case, but for simple images it usually is.